src/model/formToDats.js

```diff
@@ -32,12 +32,25 @@ class FormToDats {
         }
         delete c.type
         delete c.role
+        delete c.orcid
         if (creator.role)
           c.roles = [
             {
               value: creator.role
             }
           ]
+        if (c.type === 'Person' && creator.orcid) {
```

Comment:
Sorry, missed a bug before. `c.type === 'Person'` is never true because `c.type` is deleted on line 33. This means that the ORCID never gets added to the DATS.json.